onebox.sh

```diff
@@ -97,6 +98,10 @@ parse_args()
           -d|--default-password)
             DEFAULT_PASSWORD="${arg_value}"
             ;;
+          --msft-oauth|--msft-auth)
```

Comment:
Should |--msft-auth be removed?

Reply:
i'm good either way. it was nice having it as an alternative because i found myself dropping the "o" a bunch when testing. there are a few other places where we allow alternative options for params or their values already